app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -22,10 +30,30 @@ class Identification extends React.Component {
       current: { ...this.props.currentState },
       errors: {},
       modified: {},
-      languageOptions: this.getLanguageOptions(),
+      primaryLanguageDisplay: '',
```

Comment:
I'd like to see this be an object that holds all the info of the primary language (and likewise secondary language) including what is supported